bindata/etcd/cluster-restore.sh

```diff
@@ -53,6 +53,34 @@ function restore_static_pods() {
     tar -xvf "${BACKUP_FILE}" --strip-components=2 -C "${MANIFEST_DIR}"/ "${BACKUP_POD_PATH}"
   done
 }
+function replace_node_specific_parameters() {
```

Comment:
do we want to echo this step is taking place?

"Creating restore pod" or similar.